docs/resources.html

```diff
@@ -147,7 +147,7 @@ <h4 id="open-data-and-code">Open data and code</h4>
 				<p>For a discussion of the impact of journal data policy strictness on the code re-execution rate (i.e., how likely the code is to run without errors) and a set of recommendations for code dissemination aimed at journals, see <a href="https://arxiv.org/abs/2103.12793" target="_blank">Trisovic et al. (2020)</a>.</p>
 				<h4 id="pre-publication-verification-of-analyses">Pre-publication verification of analyses</h4>
 				<p>Some journals have now adopted a policy whereby data and code are not only required for publication in the journal, but must be checked before publication to ensure that the analyses are reproducible – that the results in the manuscript match the results that are produced when someone who is not one of the authors re-runs the code on the data. This is called pre-publication “verification of analyses”, “data and code replication”, or “reproduction of analyses”. See <a href="https://doi.org/10.1162/99608f92.25982dcf" target="_blank">Willis & Stodden (2020)</a> for a useful overview of how to leverage policies, workflows, and infrastructure to ensure computational reproducibility in publication.</p>
-				<p>For more information on how to implement a policy like this at your journal, see the <a href="https://social-science-data-editors.github.io/guidance/" target="_blank">Data and Code Guidance by Data Editors</a> developed by Lars Vilhuber and colleagues, which is used by the <a href="https://www.aeaweb.org/journals" target="_blank"><em>American Economic Association</em> journals</a>, <a href="https://onlinelibrary.wiley.com/journal/15405982" target="_blank"><em>Canadian Journal of Economics</em></a>, the <a href="https://academic.oup.com/restud" target="_blank"><em>Review of Economic Studies</em></a>, and the <a href="https://academic.oup.com/ej" target="_blank"><em>Economic Journal</em></a> as a reference.</p>
+				<p>For more information on how to implement a policy like this at your journal, see the <a href="https://social-science-data-editors.github.io/guidance/" target="_blank">Data and Code Guidance by Data Editors</a> developed by Lars Vilhuber and colleagues, which is used by the <a href="https://www.aeaweb.org/journals" target="_blank"><em>American Economic Association</em> journals</a>, <a href="https://onlinelibrary.wiley.com/journal/15405982" target="_blank"><em>Canadian Journal of Economics</em></a>, the <a href="https://academic.oup.com/restud" target="_blank"><em>Review of Economic Studies</em></a>, and the <a href="https://academic.oup.com/ej" target="_blank"><em>Economic Journal</em></a> as a reference. This  <<a href="https://labordynamicsinstitute.github.io/ldilab-manual/index.html" target="_blank">practical guide on training students to assess the reproducibility of articles in the social sciences</a> may also be of interest.</p>
```

Comment:
```suggestion
				<p>For more information on how to implement a policy like this at your journal, see the <a href="https://social-science-data-editors.github.io/guidance/" target="_blank">Data and Code Guidance by Data Editors</a> developed by Lars Vilhuber and colleagues, which is used by the <a href="https://www.aeaweb.org/journals" target="_blank"><em>American Economic Association</em> journals</a>, <a href="https://onlinelibrary.wiley.com/journal/15405982" target="_blank"><em>Canadian Journal of Economics</em></a>, the <a href="https://academic.oup.com/restud" target="_blank"><em>Review of Economic Studies</em></a>, and the <a href="https://academic.oup.com/ej" target="_blank"><em>Economic Journal</em></a> as a reference. This  <a href="https://labordynamicsinstitute.github.io/ldilab-manual/index.html" target="_blank">practical guide on training students to assess the reproducibility of articles in the social sciences</a> may also be of interest.</p>
```

Reply:
There are doubled angle-brackets `<<` in several other places -- I won't go through all of them but should be easy to search & replace